index.html

```diff
@@ -1,78 +1,83 @@
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="contact.html">Contact</a>
+                        </li>
+                    </ul>
                 </div>
             </div>
-            <div class="col-4">
-                <div class="card">
-                    <img src="./assets/woman.jpg" class="card-img-top" alt="...">
-                    <div class="card-body">
-                        <h5 class="card-title">Card title</h5>
-                        <p class="card-text">Some quick example text to build on the card title and make up the bulk of
-                            the card's content.</p>
-                        <a href="#" class="btn btn-primary">Go somewhere</a>
-                    </div>
-                </div>
+        </nav>
+    </header>
+    <main>
+        <div class="my-4 d-flex justify-content-center">
+            <img class="thumbnail max-width-4 max-height-3 w-100 mx-md-2" src="img/persona.jpg" alt="persona">        
+        </div>
+        <div class="container-xxl bd-gutter mt-3 my-md-4 bd-layout">
+            
+            <p>Je m’appelle Juliana Veiga. Je suis étudiant 
+                en développement web au cégep Garneau
+                    à Québec, depuis plus de six mois d’étude.
+            </p> 
+            <p>               
+                Je vous présente, les trois de mes 
+                projets personnels:
+            </p>
+                
+            <ul>
+                <li class="mb-2">Un petit jeu de mémoire développé 
+                    en JavaScript en utilisasant le git;</li>
+                <li class="mb-2">Une création d’un site web personnel 
+                    avec des formulaires avec html
+                    et css;</li>
+                <li class="mb-2">Une création d’un site de voyage</li>
+                <li>Voyage au Japon avec html et css et validation de formulaire.</li>
+            </ul>
+        </div>
+    </main>   
+    <footer class="p-4 bg-secondary text-dark bg-opacity-25">
+        <div class="container d-flex justify-content-between align-items-center">
+            <div>
+                <p class="mb-0">Juliana Veiga</p>
+                <p class="mb-0">Développeuse</p>
             </div>
-            <div class="col-4">
-                <div class="card">
-                    <img src="./assets/man.jpg" class="card-img-top" alt="...">
-                    <div class="card-body">
-                        <h5 class="card-title">Card title</h5>
-                        <p class="card-text">Some quick example text to build on the card title and make up the bulk of
-                            the card's content.</p>
-                        <a href="#" class="btn btn-primary">Go somewhere</a>
-                    </div>
-                </div>
+            <div>
+                <a href="https://github.com/VeigaJuliana"><img src="img/github.svg" alt="github"></a>
```

Comment:
For all your external links, try to use the `target="_blank"` property. This will make the page open in a new tab, instead to replace your current page. Do the same for all other _external_ links.